src/main/java/com/compilercharisma/chameleonbusinessstudio/entity/AppointmentEntity.java

```diff
@@ -20,108 +20,46 @@
  * 
  * @author Matt Crow <mattcrow19@gmail.com>
  */
-@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
 public class AppointmentEntity implements Serializable {
-    @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
-    @Setter
-    @Getter
-    @Column(name="appt_id")
-    private int id;
+    private  String _id;
```

Comment:
Changing from an `int` to a `String` will break a couple routes, so please make sure those still function.

Reply:
The ID in Vendia is a String

Reply:
Yeah, I know, the routing issue is just an unfortunate byproduct of that. To clarify, I'm not saying "change this back to an int", I'm saying "this change changes a few other things"